### ISSUE TITLE
NoStringInterpolationWithoutBraces does not trigger for object properties

### ISSUE THREAD

**Issue by loganhenson** [tightenco/tlint#22]
Working as intended (triggers lint)
```php
echo "$a";
```

Not working as intended (does not trigger lint)
```php
echo "$object->prop";
```
